pkgs/applications/video/mediathekview/default.nix

```diff
@@ -1,49 +1,93 @@
-{ lib, stdenv, fetchurl, makeWrapper, libglvnd, libnotify, jre, zip }:
-
-stdenv.mkDerivation rec {
-  version = "14.0.0";
+{ lib, stdenv, fetchFromGitHub
+, makeShellWrapper, makeDesktopItem, copyDesktopItems, wrapGAppsHook
+, maven, jdk
+, libglvnd, libnotify, libXxf86vm
+}:
+let
+  mavenJdk = maven.override {
+    inherit jdk;
+  };
+in
+mavenJdk.buildMavenPackage rec {
   pname = "mediathekview";
-  src = fetchurl {
-    url = "https://download.mediathekview.de/stabil/MediathekView-${version}-linux.tar.gz";
-    sha256 = "sha256-vr0yqKVRodtXalHEIsm5gdEp9wPU9U5nnYhMk7IiPF4=";
+  version = "14.0.0";
+
+  src = fetchFromGitHub {
+    owner = "mediathekview";
+    repo = "MediathekView";
+    rev = "refs/tags/${version}";
+    hash = "sha256-wVISldd5kixrCVatqI+ZUamh23Kul1QaTk+TDk55NMI=";
   };
 
+  mvnParameters = toString [
+    "-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
+    "-Dmaven.test.skip=true"
+    {
+      x86_64-linux = "-Plinux_64bit";
+      aarch64-linux = "-Plinux_arm_64bit";
+    }.${stdenv.hostPlatform.system}
```

Comment:
I guess this would fail on other platforms.  We should change meta.platforms to those two as well. Do you have an idea why the hash isn't the same? Are there sharedibs that are included into the jar?

Reply:
tbh, I haven't had a deeper look into this and I am not sure how to handle this properly for different architectures. Any suggestion welcome.

Is it acceptable to change:
```
-    platforms = lib.platforms.linux;
+    platforms = [ "x86_64-linux" "aarch64-linux" ];

```

Or completely remove `aarch64-linux`?